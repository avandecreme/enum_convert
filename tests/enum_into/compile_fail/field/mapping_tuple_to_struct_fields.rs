use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Target)]
enum Source {
    #[enum_into(Target::Struct)]
    Tuple(
        #[enum_into(Target::Struct.aa)] i32,
        #[enum_into(Target::Struct.bb)] i32,
    ),
}

enum Target {
    Struct { aa: i32, bb: i32 },
}

fn main() {}
