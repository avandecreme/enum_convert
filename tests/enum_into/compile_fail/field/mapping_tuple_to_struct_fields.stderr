error: Unexpected mapping to named field for tuple variant
 --> tests/enum_into/compile_fail/field/mapping_tuple_to_struct_fields.rs:8:36
  |
8 |         #[enum_into(Target::Struct.aa)] i32,
  |                                    ^^
